assets/models/system/tiiuae-falcon-7b/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: huggingface/tiiuae-falcon-7b/2f5c3cd/mlflow_model_folder
```

Comment:
@MallikharjunaThota123 please update the path once model is saved to blobstorage